x-pack/plugins/security_solution/public/overview/components/detection_response/host_alerts_table/host_alerts_table.tsx

```diff
@@ -91,20 +76,26 @@ export const HostAlertsTable = React.memo(({ signalIndexName }: HostAlertsTableP
               }
             />
             <EuiSpacer size="m" />
-            <EuiButton data-test-subj="severityHostAlertsButton" onClick={navigateToHosts}>
-              {i18n.VIEW_ALL_HOST_ALERTS}
-            </EuiButton>
+            {pagination.pageCount > 1 && (
+              <EuiTablePagination
+                data-test-subj="hostTablePaginator"
+                activePage={pagination.currentPage}
+                itemsPerPage={4}
```

Comment:
we don't want the user to be able to configure the per page count?

Reply:
nah, hard coded to 4. I'll make it a const variable instead.